### PR TITLE
feat(loader): add experimental colors to loader

### LINKED ISSUE #1298: Experimental Component - Loading - Development
**Development**
A developer can discover and install the experimental component by following accessible documentation.

- [ ] Add to carbon-components
- [ ] Update website (add to the experimental section and update component status page)

### PR DESCRIPTION
**Pull request by aledavila**
Closes #1298 

#### Changelog

**New**

- adds additional circle path to svg to achieve background in small variation

**Changed**

- colors

http://loading-x-anxious-meerkat.mybluemix.net/?nav=loading

### CODE REVIEW

File: src/components/loading/loading.hbs [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <div data-loading class="bx--loading{{#if small}} bx--loading--small{{/if}}">
   <svg class="bx--loading__svg" viewBox="-75 -75 150 150">
     <title>Loading</title>
-    <circle cx="0" cy="0" r="37.5" />
+    {{#if small}}<circle class="bx--loading__background" cx="0" cy="0" r="37.5" />{{/if}}
```
Comment (asudoh) [inline src/components/loading/loading.hbs]: Does this change work OK in stable mode...?

**Review by asudoh — COMMENTED** [review]
LGTM basically, just one comment - Thanks @aledavila!

**Review by alisonjoseph — APPROVED** [review]
LGTM once design (@aagonzales ) signs off

File: src/components/loading/loading.hbs (hunk 1 above)
Comment (alisonjoseph) [reply src/components/loading/loading.hbs]: Confirmed it is fine in stable mode 👍

**Review by aagonzales — APPROVED** [review]
Look good 💯

**Pull request merged**